infra/terraform/secure_pipelines/mob_async_backend_pl.tf

```diff
@@ -71,8 +71,7 @@ locals {
 
       SigningProfileArn        = one(data.aws_cloudformation_stack.signer_build[*].outputs["SigningProfileArn"])
       SigningProfileVersionArn = one(data.aws_cloudformation_stack.signer_build[*].outputs["SigningProfileVersionArn"])
-
-      TestImageRepositoryUri = one(data.aws_cloudformation_stack.mob_async_backend_tir_build[*].outputs["TestRunnerImageEcrRepositoryUri"])
+      
```

Comment:
Is there any issue here that these locals have _build in the name?

Reply:
(and the ones in 66,67)

Reply:
Its because its how they are named [here](https://github.com/govuk-one-login/mobile-id-check-async/blob/main/infra/terraform/secure_pipelines/main.tf). I think its how Terraform is able to access CFN stack outputs.